src/plots/cartesian/dragbox.js

```diff
@@ -543,8 +548,12 @@ module.exports = function dragBox(gd, plotinfo, x, y, w, h, ns, ew) {
             pushActiveAxIds(yaLinked);
         }
 
+        updates = {};
         for(i = 0; i < activeAxIds.length; i++) {
-            doTicks(gd, activeAxIds[i], true);
+            var axId = activeAxIds[i];
+            doTicks(gd, axId, true);
+            var ax = getFromId(gd, axId);
+            updates[ax._name + '.range'] = ax.range.slice();
```

Comment:
@etpinard this is significantly simpler than what I was doing before ([calculating the range edits again in dragTail](https://github.com/plotly/plotly.js/pull/1767/commits/b1c6b24ed457310c20f4651c621460348fb729d5#diff-1bd3dd52cddb8638876bbe17cef28a0dL644)) and more robust since it's explicitly WYSIWYG - whenever you update tick labels you record that and use it as the basis of the update object to be sent to `relayout`.

Zoombox has a counterpart [here](https://github.com/plotly/plotly.js/pull/1767/commits/b1c6b24ed457310c20f4651c621460348fb729d5#diff-1bd3dd52cddb8638876bbe17cef28a0dR827)

Oh, I'm noticing now that previously we sent each end of the range as a separate edit in the update object (ie `'xaxis.range[0]': 1, 'xaxis.range[1]': 5` rather than `'xaxis.range': [1, 5]`) - this way is more compact but I bet it'll break some applications that listen to relayout events to respond to zooms and inspect the event data to find the range changes, so I think I should change it back to the previous form.